lib/babe/errors.go

```diff
@@ -66,6 +66,7 @@ var (
 	errNoEpochData           = errors.New("no epoch data found for upcoming epoch")
 	errFirstBlockTimeout     = errors.New("timed out waiting for first block")
 	errChannelClosed         = errors.New("block notifier channel was closed")
+	errPrimarySlotThreshold  = errors.New("failed checking primary threshold")
```

Comment:
```suggestion
	errOverPrimarySlotThreshold  = errors.New("cannot claim slot, over primary threshold")
```

Reply:
Done in https://github.com/ChainSafe/gossamer/pull/2072/commits/5848eaff5e92c1431a2a601ca79bd8d06fe540b0